OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -47,6 +47,7 @@ public void WorldLoaded(World w, WorldRenderer wr)
 			register("all", "toggles all cheats and gives you some cash for your trouble.");
 			register("crash", "crashes the game.");
 			register("levelup", "adds a specified number of levels to the selected actors.");
+			register("radar", "toggles the minimap if you don't have an actor that provides it.");
```

Comment:
We should be consistent wrt `radar` or `minimap`.
Also, like my toggleui PR, I'd like to ask for `toggle` to be in here somewhere as that is what you're actually doing.